fix(css): document `stretch` value (#35262)
mdn/content: files/en-us/web/css/height/index.md

  - : Use the available space, but not more than [max-content](/en-US/docs/Web/CSS/max-content), i.e. `min(max-content, max(min-content, stretch))`.
- `fit-content({{cssxref("&lt;length-percentage&gt;")}})`
  - : Uses the fit-content formula with the available space replaced by the specified argument, i.e. `min(max-content, max(min-content, <length-percentage>))`.
- `stretch` {{Experimental_Inline}}

  - : Limits the height of the element's [margin box](/en-US/docs/Learn/CSS/Building_blocks/The_box_model#parts_of_a_box) to the height of its [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). It attempts to make the margin box fill the available space in the containing block, so in a way behaving similar to `100%` but applying the resulting size to the margin box rather than the box determined by [box-sizing](/en-US/docs/Web/CSS/box-sizing).

    > [!NOTE]

>     > Refer the [browser compatiblity section](#browser_compatibility) to know about the aliases used by the browsers and the current status of the implementation of the `stretch` property.

Reviewer: Looks good, thanks for adding 👍
Added suggestion to make more compact and to use "value" instead of "property".
Uh - and on second thoughts, might be better to flip it to first say the _what_ and then give the link. (also makes "aliases" the first thing readers will notice)
```suggestion
    > To check aliases used by browsers for the `stretch` value and its implementation status, see the [Browser compatibility](#browser_compatibility) section.
```

  - : Use the available space, but not more than [max-content](/en-US/docs/Web/CSS/max-content), i.e. `min(max-content, max(min-content, stretch))`.
- `fit-content({{cssxref("&lt;length-percentage&gt;")}})`
  - : Uses the fit-content formula with the available space replaced by the specified argument, i.e. `min(max-content, max(min-content, <length-percentage>))`.
- `stretch` {{Experimental_Inline}}

>   - : Height of the [margin box](/en-US/docs/Learn/CSS/Building_blocks/The_box_model#parts_of_a_box) is matched to the size of the element's [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). The `stretch` behaves as `100%` but it applies the resulting size to its margin box instead of the box indicated by [box-sizing](/en-US/docs/Web/CSS/box-sizing).

Reviewer: similar changes as suggested for `max-height`. Plus this description should begin with "Sets..." to be be parallel with other value descriptions.
